src/core/src/op/space_to_batch.cpp

```diff
@@ -102,7 +102,7 @@ bool ngraph::op::v1::SpaceToBatch::evaluate_space_to_batch(const HostTensorVecto
 
     auto data_shape = data->get_shape();
 
-    if (!(data->get_shape().size() == 4 || data->get_shape().size() == 5)) {
```

Comment:
Can we touch source code in core module directly? I think you should bring reviewer from OpenVINO core team and get confirmation.

Reply:
I invited @ilyachur as a reviewer. Thank you.

Reply:
Could you please add new tests for batch_to_space/space_to_batch to template plugin? To cover reference implementation: https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/template/tests/functional/op_reference

Reply:
Added functional tests for batch_to_space/space_to_batch to template plugin. Could you please review it?